### PR TITLE
Upgrade postgresql to 12

### PR DESCRIPTION
**Pull request by RemiDesgrange**
I did not try the code yet (dark story about capted internet) I'll try after the 9th

### CODE REVIEW

**Comment by landryb** [comment]
fwiw im using psql 12/pgis 3 in some production servers (not georchestra yet, but nextcloud/mapserver/drupal/gdal clients) and it works pretty fine, the only drawback being that you need a psql 12 client for some operations (ie at least `\d`)

**Comment by fvanderbiest** [comment]
LGTM

**Comment by fvanderbiest** [comment]
At runtime it's a different story ...
```
proxy_1                     | log4j:ERROR Failed to insert the log
proxy_1                     | java.sql.SQLException: An attempt by a client to checkout a Connection has timed out.
proxy_1                     | 	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
proxy_1                     | 	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:77)
proxy_1                     | 	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:690)
proxy_1                     | 	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
proxy_1                     | 	at org.georchestra.ogcservstatistics.dataservices.DataServicesConfiguration.getConnection(DataServicesConfiguration.java:73)
proxy_1                     | 	at org.georchestra.ogcservstatistics.log4j.OGCServicesAppender.insert(OGCServicesAppender.java:205)
proxy_1                     | 	at org.georchestra.ogcservstatistics.log4j.OGCServicesAppender.lambda$append$0(OGCServicesAppender.java:196)
proxy_1                     | 	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1640)
proxy_1                     | 	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1632)
proxy_1                     | 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
proxy_1                     | 	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
proxy_1                     | 	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
proxy_1                     | 	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
proxy_1                     | Caused by: com.mchange.v2.resourcepool.TimeoutException: A client timed out while waiting to acquire a resource from com.mchange.v2.resourcepool.BasicResourcePool@22de2f79 -- timeout at awaitAvailable()
proxy_1                     | 	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1505)
proxy_1                     | 	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
proxy_1                     | 	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
proxy_1                     | 	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
proxy_1                     | 	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
proxy_1                     | 	... 10 more
```

File: postgresql/Dockerfile [hunk 1]
```diff
@@ -2,18 +2,18 @@
 # Dockerfile for the geOrchestra database service
 #
 
-FROM postgres:11
+FROM postgres:12
 
-MAINTAINER PSC "psc@georchestra.org"
+LABEL maintainer="psc@georchestra.org"
```
Comment (RemiDesgrange) [inline postgresql/Dockerfile:7]: Because: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

**Comment by RemiDesgrange** [comment]
I cannot find any bug in master, I run tests (with IT) and launch the composition. No issue found.

File: postgresql/Dockerfile (hunk 1 above)
Comment (fvanderbiest) [reply postgresql/Dockerfile:7]: Good catch !

**Pull request merged**